tests/e2e/etcd_mix_versions_test.go

```diff
@@ -123,3 +124,133 @@ func mixVersionsSnapshotTest(t *testing.T, clusterVersion, newInstanceVersion e2
 		return true
 	}, 10*time.Second, 500*time.Millisecond)
 }
+
+func TestMixVersionsSnapshotByMockingPartition(t *testing.T) {
+	cases := []struct {
+		name                   string
+		clusterVersion         e2e.ClusterVersion
+		mockPartitionNodeIndex int
+	}{
+		{
+			name:                   "etcd instance with last version receives snapshot from the leader with current version",
+			clusterVersion:         e2e.MinorityLastVersion,
+			mockPartitionNodeIndex: 2,
+		},
+	}
+
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			mixVersionsSnapshotTestByMockPartition(t, tc.clusterVersion, tc.mockPartitionNodeIndex)
+		})
+	}
+}
+
+func mixVersionsSnapshotTestByMockPartition(t *testing.T, clusterVersion e2e.ClusterVersion, mockPartitionNodeIndex int) {
+	e2e.BeforeTest(t)
+
+	if !fileutil.Exist(e2e.BinPath.EtcdLastRelease) {
+		t.Skipf("%q does not exist", e2e.BinPath.EtcdLastRelease)
+	}
+
+	// Create an etcd cluster with 3 member of MinorityLastVersion
+	epc, err := e2e.NewEtcdProcessCluster(context.TODO(), t,
+		e2e.WithClusterSize(3),
+		e2e.WithSnapshotCount(10),
+		e2e.WithVersion(clusterVersion),
+		e2e.WithSnapshotCatchUpEntries(10),
+	)
+	if err != nil {
+		t.Fatalf("failed to start etcd cluster: %v", err)
+	}
```

Comment:
```suggestion
	require.NoError(t, err)
```

Please use `github.com/stretchr/testify/require` to simplify both test cases: `TestMixVersionsSnapshotByMockingPartition` and `TestMixVersionsSnapshotByAddingMember`

Reply:
Note this comment applies to multiple places in both test cases.

Reply:
done